### ISSUE TITLE
AttributeError: 'Namespace' object has no attribute 'json'

### ISSUE THREAD

**Issue by helloyw** [ly4k/Certipy#18]
Please tell me, my current environment is python3.9, when executed, `Certipy 'test/join:PassWo!rd@dc.test.local' auto,` the following error appears, is there any solution?

Traceback (most recent call last):
  File "/usr/local/bin/certipy", line 33, in <module>
    sys.exit(load_entry_point('Certipy==0.2', 'console_scripts', 'certipy')())
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/entry.py", line 176, in main
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/auto.py", line 29, in auto
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/find.py", line 474, in __init__
AttributeError: 'Namespace' object has no attribute 'json'

**Comment by mxrch** [comment]
Got the same error with Python 3.9.

**Comment by mxrch** [comment]
As a temporary workaround, you can just comment these two lines at certipy/find.py:474 :
```python
        #if self.options.json:
        #    logging.getLogger().setLevel(logging.WARNING)
```
They are only logger-related and not necessary.

**Comment by helloyw** [comment]
Thank mxrch！

**Comment by helloyw** [comment]
> 作为临时解决方法，您可以在 certipy/find.py:474 注释这两行：
> 
> ```python
>         #if self.options.json: 
>         # logging.getLogger().setLevel(logging.WARNING)
> ```
> 
> 它们仅与记录器相关，而不是必需的。

Thank mxrch

**Comment by helloyw** [comment]
After these two lines have been commented, execute the command `Certipy 'test/join:PassWo!rd@dc.test.local' auto` again, and a new error occurs
`Traceback (most recent call last):
  File "/usr/local/bin/certipy", line 33, in <module>
    sys.exit(load_entry_point('Certipy==0.2', 'console_scripts', 'certipy')())
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/entry.py", line 176, in main
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/auto.py", line 30, in auto
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/find.py", line 495, in connect
  File "/usr/local/lib/python3.9/dist-packages/Certipy-0.2-py3.9.egg/certipy/ldap.py", line 72, in connect
  File "/usr/local/lib/python3.9/dist-packages/impacket-0.9.25.dev1+20220105.151306.10e53952-py3.9.egg/impacket/ldap/ldap.py", line 122, in __init__
    self._socket.do_handshake()
  File "/usr/lib/python3/dist-packages/OpenSSL/SSL.py", line 1894, in do_handshake
    self._raise_ssl_error(self._ssl, result)
  File "/usr/lib/python3/dist-packages/OpenSSL/SSL.py", line 1624, in _raise_ssl_error
    raise SysCallError(errno, errorcode.get(errno))
OpenSSL.SSL.SysCallError: (104, 'ECONNRESET')
`

**Comment by ly4k** [comment]
Fixed in #91cbec8
